pkg/grpc/grpc.go

```diff
@@ -159,7 +157,7 @@ func (g *Manager) createLocalConnection(parentCtx context.Context, port int, ena
 	}
 
 	dialPrefix := GetDialAddressPrefix(g.mode)
-	address := net.JoinHostPort(g.channelConfig.BaseAddress, strconv.Itoa(port))
+	address := net.JoinHostPort("127.0.0.1", strconv.Itoa(port))
```

Comment:
DefaultChannelAddress?

Reply:
This needs to be `address := net.JoinHostPort(g.channelConfig.BaseAddress, strconv.Itoa(port))`.

Reply:
thanks for catching this :)